### PR TITLE
🦃 rbac for argocd bootstrap 🦃

### PR DESCRIPTION
**Pull request by eformat**
See: https://github.com/rht-labs/helm-charts/pull/68

adding namespace RBAC for ArgoCD service account

### CODE REVIEW

File: bootstrap/values-bootstrap.yaml [hunk 1]
```diff
@@ -59,6 +59,14 @@ argocd-operator:
     prometheus:
       version: prometheusoperator.0.32.0
 
+  # argocd rbac only in listed namespaces
+  namespaceRoleBinding:
+    enabled: true
+    namespaces:
+    - name: *ci_cd
+    - name: *dev
+    - name: *test
```
Comment (springdo) [inline bootstrap/values-bootstrap.yaml:68]: Our ArgoCD app is probs going to only manage things in `labs-test` and `labs-staging` right? Maybe those could be better defaults

File: bootstrap/Chart.yaml [hunk 2]
```diff
@@ -11,8 +11,8 @@ dependencies:
     version: "0.0.6"
     repository: https://rht-labs.github.io/helm-charts
   - name: argocd-operator
-    version: "0.0.13"
-    repository: https://rht-labs.github.io/helm-charts
+    version: "0.0.14"
+    repository: https://eformat.github.io/helm-charts
```
[the remark this answers is not in the document]
Comment (eformat) [reply bootstrap/Chart.yaml:15]: 🦜doh! thanks 🦜

File: bootstrap/values-bootstrap.yaml [hunk 3]
```diff
@@ -59,6 +59,14 @@ argocd-operator:
     prometheus:
       version: prometheusoperator.0.32.0
 
+  # argocd rbac only in listed namespaces
+  namespaceRoleBinding:
+    enabled: true
```
[the remark this answers is not in the document]
Comment (eformat) [reply bootstrap/values-bootstrap.yaml:64]: 🦜oops, testing .. no we do not want to enable this 🦜

File: bootstrap/values-bootstrap.yaml (hunk 1 above)
Comment (eformat) [reply bootstrap/values-bootstrap.yaml:68]: it needs all the projects that the argocd service account will need .. i will add staging as well.

it needs *ci_cd for sure - that where the UJ apps live
it needs *dev - else you wont be able to interact from argo to here for YOUR projects apps so seems sane to include

**Pull request merged**